scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -160,15 +128,56 @@ private void changeZValue(){
     double zValue = zValueSlider.getValue();
 
     // update the geometry of each of the existing graphics to include the new z-value
-    for (GraphicsOverlay graphicsOverlay : sceneView.getGraphicsOverlays()) {
-      for (Graphic graphic : graphicsOverlay.getGraphics()) {
+    sceneView.getGraphicsOverlays().forEach(graphicsOverlay ->
+      graphicsOverlay.getGraphics().stream().filter(
+        graphic -> graphic.getGeometry().getGeometryType() == GeometryType.POINT).forEach(graphic -> {
         Geometry geometry = graphic.getGeometry();
-        if (geometry.getGeometryType().equals(GeometryType.POINT)) {
-          Point currentPoint = (Point) geometry;
-          Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
-          graphic.setGeometry(updatedPoint);
-        }
-      }
+        Point currentPoint = (Point) geometry;
+        Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
+        graphic.setGeometry(updatedPoint);
+      }));
+  }
+
+  /**
+   * Creates a new TextSymbol from a string used to identify a surface placement type.
+   * @param text string to be used as text within the TextSymbol constructor
+   * @return a new TextSymbol
+   *
+   */
+  private TextSymbol createTextSymbol(String text) {
+
+    TextSymbol textSymbol = new TextSymbol(14, text, 0xFF0000FF,
+      TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
+    textSymbol.setOffsetX(20);
+
+    if (text == "RELATIVE TO SCENE") {
```

Comment:
I'm getting an IDE suggestion here to replace the `==` operator with `.equals`.  The latter method checks the actual contents of the string, whereas the former checks whether the references to the objects are equal.